### PR TITLE
Update fr.po

### PR DESCRIPTION
**Pull request by miiPoP**
Update french translation in sync with WordDumb v3.26.0

### CODE REVIEW

File: translations/fr.po [hunk 1]
```diff
@@ -93,258 +89,166 @@ msgstr ""
 msgid "Source code and document"
 msgstr "Code source et document"
 
-#: config.py:207 config.py:224 config.py:279
+#: config.py:207 config.py:224 config.py:275
 msgid "Saving customized lemmas"
 msgstr "Sauvant les lemmes personalisés"
 
-#: config.py:243
```
Comment (xxyzz) [inline translations/fr.po:100]: Line 100 to 190 need to be preserved for the EPUB Word Wise language drop down button, these texts are not hard coded in the Python file so Poedit always moves them commented to the end when clicking the "Update from source code" button.

File: translations/fr.po [hunk 2]
```diff
@@ -93,258 +89,166 @@ msgstr ""
...
+#: config.py:441
+msgid "Lemma language"
+msgstr "Langue des lemmes"
 
-#: custom_lemmas.py:57
-msgid "Kindle Word Wise"
-msgstr "Word Wise Kindle"
+#: config.py:447
+msgid "Gloss language"
+msgstr "Langue du Gloss (Global Language Online Support System)"
```
Comment (xxyzz) [inline translations/fr.po:137]: "gloss" is not an acronym, it's just the short definition of a word(lemma).

File: translations/fr.po [hunk 3]
```diff
@@ -627,3 +531,84 @@ msgstr "Générant {} pour {}"
 #: ui.py:126
 msgid "{} generated for {}"
 msgstr "{} généré pour {}"
+
```
Comment (xxyzz) [inline translations/fr.po:534]: This chunk of commented change is unnecessary as mentioned before.

**Review by xxyzz — CHANGES_REQUESTED** [review]
Hi, thanks for your pull request! I add some comments about the changes, please take a look.

File: translations/fr.po (hunk 1 above)
Comment (miiPoP) [reply translations/fr.po:100]: Oops thought Poedit updating from source was safe. Lets uncomment these then.

File: translations/fr.po (hunk 2 above)
Comment (miiPoP) [reply translations/fr.po:137]: "Langue du gloss" OK then?

File: translations/fr.po (hunk 3 above)
Comment (miiPoP) [reply translations/fr.po:534]: I don't understand this one, doesn't seem :126  is commented.

File: translations/fr.po (hunk 2 above)
Comment (xxyzz) [reply translations/fr.po:137]: Sure.

File: translations/fr.po (hunk 3 above)
Comment (miiPoP) [reply translations/fr.po:534]: Got it.  It is lines 534-614 that are commented (and must be preserved). Just don't understand Poedit behaviour. Looking at config.py I remain clueless.

File: translations/fr.po (hunk 2 above)
Comment (miiPoP) [reply translations/fr.po:137]: Correct. Glose is a better translation. I learned a new word today :-)

File: translations/fr.po (hunk 3 above)
Comment (xxyzz) [reply translations/fr.po:534]: These language names are used at here:
https://github.com/xxyzz/WordDumb/blob/316e19fba35ff678065b5284d9a2872e398d3c87/config.py#L422-L429

`lemma_languages` and `gloss_languages` are used for the drop down button text at here:
https://github.com/xxyzz/WordDumb/blob/316e19fba35ff678065b5284d9a2872e398d3c87/config.py#L438-L445

Comment (miiPoP) [reply translations/fr.po:534]: Ah, I see. Thanks for explaining.

**Pull request closed without merging**